packages/cli/src/eventbus/MessageEventBus/MessageEventBus.ts

```diff
@@ -41,6 +42,7 @@ export interface MessageWithCallback {
 export interface MessageEventBusInitializeOptions {
 	skipRecoveryPass?: boolean;
 	workerId?: string;
+	replicateToRedisEventLogFunction?: (msg: AbstractEventMessage) => Promise<void>;
```

Comment:
I think given the current approach we're not relaying the messages to Redis, right? Is this in the scope of this PR?